src/components/modes/student/StudentView.js

```diff
@@ -16,7 +16,7 @@ const styles = (theme) => ({
   },
 });
 
-export const StudentView = () => <CodeReview />;
+export const StudentView = () => <CodeReview isStudentView />;
```

Comment:
I think it could be best to pass a `view` prop and do a switch or write a method in the CodeReview component. Just for reusability's sake in case we need something special for Teacher. You can also omit this completely and get the mode prop directly from the context reducer.

Reply:
Yes, your suggestion makes sense, I will go with the "view" prop as i have other places where i pass in flags like `isTeacherView` or `isFeedbackView`. Even if in this case, the alternative with using the mode from the context prop would maybe also make sense because it is explicitly related to the way it is displayed inside the iframe and not really to some internal state of the component.